source/plugins/go/src/telemetry.go

```diff
@@ -297,6 +312,20 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 				telemetryDimensions["PromFbitBufferSize"] = os.Getenv("AZMON_FBIT_BUFFER_SIZE")
 				telemetryDimensions["PromFbitMemBufLimit"] = os.Getenv("AZMON_FBIT_MEM_BUF_LIMIT")
 
+				if IsNetworkFlowLogsEnabled {
+					telemetryDimensions["NetworkFlowLogsEnabled"] = "true"
+					IsNetworkFlowLogsThrottleEnabled := os.Getenv("NETWORKFLOW_LOGS_THROTTLE_ENABLED")
+					if IsNetworkFlowLogsThrottleEnabled != "" && strings.EqualFold(IsNetworkFlowLogsThrottleEnabled, "true") {
+						telemetryDimensions["NetworkFlowLogsThrottleEnabled"] = IsNetworkFlowLogsThrottleEnabled
```

Comment:
For optimizing the telemtry instead of NetworkFlowLogsThrottleEnabled, we can have something like "nflThrottleEnabled".

Reply:
changed